### PR TITLE
[FIX] allow superuser to set log_access fields

### PR DESCRIPTION
**Pull request by rco-odoo**
Goal: enable a module to create data in the past.
For the sake of consistency, also make the log_access fields readonly.

### CODE REVIEW

**Comment by rco-odoo** [comment]
@RomainLibert

**Comment by odony** [comment]
Based on further discussion, this is driven by the urge to set past date in demo/module data, for the few cases where the log_access fields are used in business context too (e.g Lead creation date).

A compromise to support this use case without losing auditing properties would be to allow setting the log_access fields only during module installation (while the registry is still loading).

**Comment by rco-odoo** [comment]
ping @odony modified just like we decided

File: addons/project/models/project.py [hunk 1]
```diff
@@ -451,8 +451,8 @@ def _read_group_stage_ids(self, stages, domain, order):
         ('blocked', 'Red')], string='Kanban State',
         copy=False, default='normal', required=True)
     kanban_state_label = fields.Char(compute='_compute_kanban_state_label', string='Kanban State Label', track_visibility='onchange')
-    create_date = fields.Datetime(index=True)
-    write_date = fields.Datetime(index=True)  #not displayed in the view but it might be useful with base_automation module (and it needs to be defined first for that)
+    create_date = fields.Datetime("Created On", readonly=True, index=True)
```
Comment (mreficent) [inline addons/project/models/project.py]: why in those cases not just putting `index=True`? The string and the `readonly` attributes will be inherited from the base model, right? 🤔

Comment (rco-odoo) [reply addons/project/models/project.py]: @mreficent the log_access fields are **not** on the base model. The reason is that you can choose to not have them by setting `_log_access = False` on your model's class.

In practice, the automatic field `create_uid` (resp. `create_date`, `write_uid`, `write_date`) is added to the model's final class only if it is not defined yet.

Comment (mreficent) [reply addons/project/models/project.py]: Thanks for your answer :)

**Pull request merged**